packages/plugins/src/plugins/morpho/index.ts

```diff
@@ -1,11 +1,26 @@
+import { NetworkId } from '@sonarwatch/portfolio-core';
 import { Fetcher } from '../../Fetcher';
 import { Job } from '../../Job';
 import aaveV3CollateralFetcher from './aaveV3CollateralFetcher';
 import aaveV3SupplyOnlyFetcher from './aaveV3SupplyOnlyFetcher';
 import aaveV3Job from './aaveV3Job';
 
-export const jobs: Job[] = [aaveV3Job];
+import { getLendingFetcher } from './lendingFetcher';
+import { getYieldFetcher } from './yieldFetcher';
+import { getRewardsFetcher } from './rewardsFetcher';
+
+import { marketsJob } from './marketsJob';
+import { vaultsJob } from './vaultsJob';
+
+export const jobs: Job[] = [
+  aaveV3Job,
+  marketsJob(NetworkId.ethereum),
+  vaultsJob(NetworkId.ethereum),
+];
 export const fetchers: Fetcher[] = [
   aaveV3CollateralFetcher,
   aaveV3SupplyOnlyFetcher,
+  getLendingFetcher(NetworkId.ethereum),
+  getYieldFetcher(NetworkId.ethereum),
+  getRewardsFetcher(NetworkId.ethereum),
```

Comment:
Did you see examples where _factories_ were used ?

Something along the lines of:

```suggestion
  LendingFetcherFactory.forNetwork(NetworkId.ethereum),
  YieldFetcherFactory.forNetwork(NetworkId.ethereum),
  RewardsFetcherFactory.forNetwork(NetworkId.ethereum),
```

The concept ends up being the same, but the naming suggests that the factory might bring an additional value (think validation of parameters). The factory pattern is often used with `Value Objects` too (https://en.wikipedia.org/wiki/Value_object -- although the code examples on Wiki are showing the use of constructor).

Ref: https://en.wikipedia.org/wiki/Factory_method_pattern

Let me know what you think.

Reply:
You can ignore the above comment; @BitWonka provided insight on how the project is built (fetchers vs executors).